content/millennium/dstu2/encounters.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Encounters
+title: Encounters | DSTU 2 API
 layout: api
 ---
 
-<%= render '/encounters.*' %>
```

Comment:
Why did this need to get qualified? The dstu2 is a folder?

Reply:
I renamed the existing one to encounters_dstu2 and made a new one for encounters_r4 https://github.com/cerner/fhir.cerner.com/pull/237/files#diff-3bea4943c34362eafcffdc1a5dc29b5a

that seemed simpler than flexing the link inside the .html file - and also the dstu2 links don't seem to have exact r4 counterparts

Reply:
discussed offline:

* changed template names back to just encounters and just individuals
* conditionally display resourceguide link only when viewing the overview in the context of dstu2

81ac133